exec/java-exec/src/test/java/org/apache/drill/exec/physical/impl/TestSimpleFunctions.java

```diff
@@ -62,10 +62,11 @@
 
 public class TestSimpleFunctions extends ExecTest {
   //private static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(TestSimpleFunctions.class);
-  private final DrillConfig c = DrillConfig.create();
+//  private final DrillConfig c = DrillConfig.create();
```

Comment:
I can remove it. I initially commented it just to see if things would still work.